DATA/tools/epn/gen_topo.sh

```diff
@@ -26,17 +26,13 @@ fi
 [[ -z "$GEN_TOPO_EPN_CCDB_SERVER" ]] && export GEN_TOPO_EPN_CCDB_SERVER="http://127.0.0.1:8084" # CCDB server to use
 if [[ "0$GEN_TOPO_ONTHEFLY" == "01" ]]; then export SHM_MANAGER_SHMID=1 ;fi
 
-# Command for topology generation
-if [[ -z "$GEN_TOPO_ODC_EPN_TOPO_CMD" ]]; then
-  export GEN_TOPO_ODC_EPN_TOPO_CMD='/home/lkrcal/epn/topogen/.venv/bin/python3 /home/lkrcal/epn/topogen/topo_merger.py'
-fi
+# Command for topology merging
+export GEN_TOPO_ODC_EPN_TOPO_CMD='/usr/local/bin/epn-topo-merger'
+
 # Command for postprocessing of topology generation after topology caching
-export GEN_TOPO_ODC_EPN_TOPO_POST_CACHING_CMD='/home/lkrcal/epn/topogen/.venv/bin/python3 /home/lkrcal/epn/topogen/topo_resource_alloc.py'
+export GEN_TOPO_ODC_EPN_TOPO_POST_CACHING_CMD='/usr/local/bin/epn-topo-alloc'
```

Comment:
Could you please keep the `if [[ -z...` and we should have an `if [[ -z...` for GEN_TOPO_ODC_EPN_TOPO_POST_CACHING_CMD as well.

Reply:
I was hoping that since there was no defaulting for GEN_TOPO_ODC_EPN_TOPO_POST_CACHING_CMD, we are finally getting rid of all these env variables. :-)